mixins/BaseMixin.ts

```diff
@@ -45,6 +41,38 @@ export default class BaseMixin extends Vue {
             : this.$config.profileId.managed_cards_corporates!
     }
 
+    get identityRegCountryIsUK() {
+        return this.isConsumer ? this.stores.consumers.isUk : this.stores.corporates.isUk
+    }
+
+    get cardJuristicationProfileId() {
```

Comment:
Very minor but I think you have a typo in Jurisdiction

Reply:
sorted. nice one